Updated notebook for compatibility with refactor

radelja opened the pull request:
Fixed issues with the run configuration output, and updated notebook files for compatibility with the refactor.

radelja committed:
Updated output of run configs

radelja committed:
Add GettingStarted.ipynb and cpu-stream.json from commit 8aadf5abdab1968f82c08f289da1b1c596553eda

radelja committed:
Uniform stride suite now works with refactor branch

radelja committed:
Fixed incorrect kernel map value

JDTruj2018 commented on standard-suite/basic-tests/cpu-stream.json:
Do we want to keep the "pattern-size" parameter in the test which truncates the pattern from a length of 8 to 4?

radelja commented:
Before I changed cpu-stream.json on the main branch, the notebook was setup to check Gather, Scatter, GS, MultiGather, and MultiScatter. At the time, the file did not reflect this and the notebook was failing when the system type was set to 'cpu'. When I updated it for compatibility with the notebook, I modified it to mirror the [gpu-stream.json](https://github.com/JDTruj2018/spatter/blob/refactor/standard-suite/basic-tests/gpu-stream.json) file.

Since you've raised this concern, I'll add the 'pattern-size' parameter back while maintaining similarity with the format used by gpu-stream.json.

radelja committed:
Added pattern-size argument back to cpu-stream.json
